homeassistant/components/camera/local_file.py

```diff
@@ -61,7 +80,23 @@ def camera_image(self):
             _LOGGER.warning("Could not read camera %s image from file: %s",
                             self._name, self._file_path)
 
+    def update_file_path(self, file_path):
+        """Update the local_file camera file path."""
+        if os.path.isfile(file_path):
+            self._file_path = file_path
```

Comment:
I think you need to call `self.schedule_update_ha_state()` (or something like that) here to push the state change to the state machine 😉

Reply:
@OttoWinter thanks for the suggestion, the camera image on the front end now updates instantly, where previously it took several seconds to update :)

However my test is still failing with the error:
```
Could not read camera Local File image from file: mock/path.jpg
``` 
Any ideas? @dgomes ?